docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,374 @@
+
+### Where can I store the data?
+
+What to consider:
+
+* Raw vs intermediate vs final result data
+    * What needs to be stored?
+    * Storage space?
+* Accessibility
+    * Sensitive data?
+    * Who needs to have access?
+    * How needs the data to be accessed?
+    * Intended usage?
+* Maintenance needs?
+* Metadata needs?
+
+See also CSC's [guide on how to choose a suitable storage solution](../../../data/datasets/hosting-datasets-at-CSC.md).
+    
+* In general, data can be stored on the [supercomputer](../../../computing/disk.md) or in the object storage Allas ([Allas overview](https://research.csc.fi/-/allas), [Allas guide](../../../data/Allas/index.md)) 
+* On the supercomputer, data can be stored on ```/scratch/project_xxx``` with xxx being your project number
+* Smaller amounts can also be stored short term on the computing nodes ```$LOCAL_SCRATCH``` during processing
+* In Allas, data is stored in so-called buckets, and can be accessed or transferred as part of the computing job, see also CSC's webinar on [Allas for spatial data](https://www.youtube.com/watch?v=mnFXe2-dJ_g).
+
+
+> For longer term storage and publication, CSC offers a range of other services. See also CSC's general [guide on stroing data](../../../data/datasets/hosting-datasets-at-CSC.md).
+
+
+### How can I process the data?
+
+At CSC, EO data can be processed and analyzed using for example supercomputer [Puhti](../../../computing/systems-puhti.md) or a virtual machine in the CSC cloud = [cPouta](../../../cloud/pouta/pouta-what-is.md). You can find more information around geocomputing using CSC resources on our [Geocomputing pages](https://research.csc.fi/geocomputing).
+
+Puhti has a lot of applications ready installed (see below), you do not need to worry about it. You can also add your own installations using for example the [Tykky tool](../../../computing/containers/tykky.md). In cPouta, you need to set up your own virtual machine including all security and software setup, see [instructions](../../../cloud/pouta/launch-vm-from-web-gui.md).
+
+#### Software
+
+What to consider:
+
+* User skills and preferences
+    * Graphical User Interface (GUI)
+    * Command Line Interface (CLI)
+    * Scripting
+* User needs
+    * Batch processing
+    * Automation
+    * Reproducibility
+* Open source vs commercial
+    * This guide focusses on [software available on CSC supercomputer Puhti](../../../apps/index.md) 
+
+##### What applications are available on Puhti?
+
+* Only Linux software
+* Mostly open source
+
+GUIs of software available on Puhti can be accessed as an interactive job via the [Puhti web interface](https://puhti.csc.fi) or [X11 connection](../../../computing/connecting.md). These graphical interfaces are mainly for visualization and testing purposes, the actual efficient processing should be done within batch  jobs rather than interactive jobs.
```

Comment:
```suggestion
GUIs of software available on Puhti can be accessed as an interactive job via the [Puhti web interface](https://puhti.csc.fi) or [X11 connection](../../../computing/connecting.md). These graphical interfaces are mainly for visualization and testing purposes, the actual efficient processing should be done within batch jobs rather than interactive jobs.
```